### ISSUE TITLE
bin/magento indexer:reindex vsbridge_product_category gives no response

### ISSUE THREAD

**Issue by mmularski** [DivanteLtd/magento2-vsbridge-indexer#184]
If you run `php bin/magento indexer:info` command, there will be a list of available indexers:
```
.
.
.
vsbridge_product_indexer                 Vsbridge Product Indexer
vsbridge_category_indexer                Vsbridge Category Indexer
vsbridge_attribute_indexer               Vsbridge Attributes Indexer
vsbridge_product_category                Vsbridge Product Category Indexer
vsbridge_cms_block_indexer               Vsbridge Cms Block Indexer
vsbridge_cms_page_indexer                Vsbridge Cms Page Indexer
vsbridge_review_indexer                  Vsbridge Review Indexer
vsbridge_taxrule_indexer                 Vsbridge Tax Rule Indexer
.
.
.
```

but if I type
```
root@58e2162df79d:/var/www/html# php bin/magento indexer:reindex vsbridge_product_category
root@58e2162df79d:/var/www/html# 
```

there is no response which causes confusion 

IMHO this indexer should be cut off from `php bin/magento indexer:info` response

edit: also in `php bin/magento indexer:reindex` supported types hint

**Comment by afirlejczyk** [comment]
Hi, 
Strange it worked for me.. I didn't modify Magento `indexer:reindex` command at all.
If you take a look into Magento command, you will notice magento only execute "valid" indexers.
So If you don't see any message that means your indexer has invalid status and wasn't executed.

```
app@4da414b8145a:~/html$ php bin/magento indexer:reindex vsbridge_attribute_indexer
Vsbridge Attributes Indexer index has been rebuilt successfully in 00:00:00
```

It's more like a bug in Magento Core, they should show us some information about this.

**Comment by mmularski** [comment]
problem is only with `vsbridge_product_category` because as I assumed it is only "On Schedule" partial indexing indice.

**Comment by afirlejczyk** [comment]
Yes, I just check this. It was done by purpose. It's only used for partial indexing and meant to be use only on category save.

I probably should hide this index in Magento Panel /  show some information why you can't run this index.

**Comment by mmularski** [comment]
Yes. I just wanted to let you know because the client informed us that it is confusing and "probably issue", so I think it is worth to consider how to hide it accurately :)

**Comment by afirlejczyk** [comment]
OK, I get it.  :)
Yes it should be hidden in Magento Panel, just didn't have time to work on this.

**Comment by afirlejczyk** [comment]
PR will be more than welcome!

**Comment by mmularski** [comment]
Soon I will try to find some time :)

**Comment by afirlejczyk** [comment]
Indexer will be now hide in magento backend #204

**Comment by afirlejczyk** [comment]
TODO:

> IMHO this indexer should be cut off from php bin/magento indexer:info response

**Comment by afirlejczyk** [comment]
- indexer is hidden in Magento panel
- indexer is not visible in commands: `indexer:info`, `indexer:reindex`

I plan to remove this "vsbridge_product_category" indexer all together.

**Comment by pcartel** [comment]
<img width="180" alt="Screen Shot" src="https://user-images.githubusercontent.com/70662862/101523580-c9c47c00-3956-11eb-91bc-1234e560e0a9.png">

My menu doesn't show any of the categories but in the search bar I can see categories and brands really weird. Probably a small issue but if you could have a look and possibly point me in the right direction it would help tremendously. On the backend of magento 2 it also says category needs reindex as well tried and nothing
